core/src/main/java/hudson/util/ChunkedInputStream.java

```diff
@@ -342,9 +343,8 @@ public void close() throws IOException {
      */
     static void exhaustInputStream(InputStream inStream) throws IOException {
         // read and discard the remainder of the message
-        byte buffer[] = new byte[1024];
+        byte[] buffer = new byte[1024];
         while (inStream.read(buffer) >= 0) {
-            ;
```

Comment:
maybe this was to comply with findbugs

Reply:
I'm not sure, I'm pretty sure we're inconsistent. I think that if we're trying to comply w/ a given tool we should always include a comment to help others know that.